### DISCUSSION TITLE
Cordova commands in capacitor/react

### DISCUSSION THREAD

**Discussion by uhotspot4** [ionic-team/capacitor#3299] [Q&A] [answered]
As has been mentioned in [https://github.com/cordova-rtc/cordova-plugin-iosrtc/blob/master/docs/Building.md#apple-store-submission](url) in order to submit the app to App Store we need to strip simulator (i386/x86_64) archs from WebRTC binary and as the very last step to do so it says:

Remove ios cordova platform if already added and add ios platform again (e.g. with a command **_cordova platform remove ios_** && **_cordova platform add ios_**) or remove and add only the plugin at your own risk.

I have a capacitor/react project in which I am using the above mentioned cordova plugin.

How can I invoke such command in a capacitor/react project or is there any equivalent steps I can take to get the same result?

**Comment by jcesarmobile** [comment, marked as answer]
Those are cordova instructions, for Capacitor just run `npx cap update ios` or `npx cap sync ios`

**Reply by uhotspot4** [reply]
@jcesarmobile Thanks for the reply. I have tried all those many times but unfortunately I still get the following message when I want to run it on a real device! d: warning: Could not find or use auto-linked framework 'WebRTC' Undefined symbols for architecture arm64: "_OBJC_CLASS_$_RTCMediaConstraints", referenced from:.....

But when I run `node ios_arch.js --list` it lists the arm7 and arm64 arches. So I guess it should only be the very last step that is not taking effect!

I would really appreciate if you can help me with this because I'm stocked at this point and frustrated! Let me know if you need any further info that might help you please.

**Reply by jcesarmobile** [reply]
But do you get the error any time you try to run on a real device? or only when you try to run on a device after you ran the node script?

Because that error you get looks like the one you get when bitcode is not disabled, see https://github.com/cordova-rtc/cordova-plugin-iosrtc/blob/master/docs/Building.md#cocoapods for how to disable it

**Reply by uhotspot4** [reply]
I will check the link out but as the answer to your question it always happens when I want to run it on a device. It works fine if I run it in simulator but doesn't work at all on a real device!

**Reply by uhotspot4** [reply]
OK thanks a lot. Those errors are gone but now it gives me a new error:

ld: '/Users/devUser/Library/Developer/Xcode/DerivedData/App-ewymbzgwftskdjhvbhyerfmlkzmxr/Build/Products/Debug-iphoneos/Capacitor/Capacitor.framework/Capacitor' does not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. file '/Users/devUser/Library/Developer/Xcode/DerivedData/App-ewymbzgwftskdjhvbhyerfmlkzmxr/Build/Products/Debug-iphoneos/Capacitor/Capacitor.framework/Capacitor' for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

**Reply by jcesarmobile** [reply]
You also have to disable bitcode for your app target

![Captura de pantalla 2020-07-21 a las 19 45 12](https://user-images.githubusercontent.com/1637892/88088456-b755cb80-cb8a-11ea-845c-f0918291170f.png)

**Reply by uhotspot4** [reply]
That worked, I really appreciate your help. Thank you
